### ISSUE TITLE
Unable to open terminal after connecting to a cluster

### ISSUE THREAD

**Issue by aaraodeo** [lensapp/lens#687]
**Describe the bug**
After connection to a cluster if we open a terminal its not opening and an error alert is being displayed.

**To Reproduce**
1. Go to 'Cluster (docker-desktop).'
2. Click on 'Terminal'
3. See error

Error
A JavaScript error occurred in the main process
Uncaught Exception:
Error: EPERM: operation not permitted , open
'C:\Users\user\AppData\Roaming\Lens\binaries\kubectl\1.16.8\kubectl.exe'

**Expected behavior**
Terminal should open fine with kubectl set to current context

**Environment (please complete the following information):**
- Lens Version: 3.5.3
- OS: Windows 10 Pro 64 bit
- Installation method (Windows binary): Lens-Setup-3.5.3.exe
- Kubernetes Env: V1.16.6 comes with docker desktop v2.3.0.4
 

**Kubeconfig:**
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN5RENDQWJDZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJd01EZ3dOVEEyTlRRMU4xb1hEVE13TURnd016QTJOVFExTjFvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTHNaCjIzRGRiZ2JndDg2bG9IYTI2YmMycmhiWnp3cllMbzczaVdyamkxTkM5VlB2R0dMQlAybjRXRnFnWDRMcG1ZSWQKODA3dXdxYXR5ZHBUWkNCR0ZOOGpVTWZaQmhMSk93bDNoZnprTmd2bE1aK1ZKbkVtN0dmbjJlNHpKaUsrRTV3cQpDSVVRL1luWUNBU2tubzVadXVoWEJUZHV0WFEva05DOU5QUXYxNlVBN203U0JUNmxQKzJkeWNWYnZMUnB2OVNpCjk3cHNqWHNRK1VmMHpRbkNJZ2J3Skh5b1RBOE9EejRXaUtXYks4NksvQjkrZm10T1VxZkdFWWlycDUvbzlwb3oKSU11M0ZjbUdJV2tOQnhJVlg0WUcyT0hRcmdqWHM0Uk9va1lGTVp3aXZaMGdkcjVKZ05Gc21ncmppamdpem9JQwpaS1JGUitHZ3ZJTUowSnBidWxzQ0F3RUFBYU1qTUNFd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFDelZEdjByZFRZcXFtdU1hT2RWVUl5T0NYRksKd2NCWDhxWUlxaWFGZFZqWStJcHVBdlpKNEtFeDlMQVd1Q2E5T094RWxCbmVWSS9aNUpvZWYvbmVvWmlpL2JKaQoxeVltd1Y4NXByTzVHOExYQUd5Ly93WWgzZnAxNzMreWV3RElpdDNjbWdiRWZ6ZG1EZ1dkYWw4L3hCM0sxNnlMCi9TRXlnN1N5RmtGSm9xUVUxbzR3RkN3TFU1K1VVbEZwTlYxUU9PWjVCQ1Q5eTlhU3ZNQ3NpUlBML0ppdjVUcFEKc0FoRytHWVp5cERzZFRBdlJvWC9QYlFNQUJBd21CMWpGbTBRU1BvUXM3S3dxa0Y1NEorYjdUSXEvQ1VyK3dFdgprTHpUWFFUNW5kejg5aEhDdXJNaGRja1RiQkVzVW0waWd5M3hDMklkbTBKU0h3cjN5Y0Jma1Qydy9jWT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
    server: https://kubernetes.docker.internal:6443
  name: docker-desktop
contexts:
- context:
    cluster: docker-desktop
    user: docker-desktop
  name: docker-desktop
current-context: docker-desktop
kind: Config
preferences: {}
users:
- name: docker-desktop
  user:
    client-certificate-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM5RENDQWR5Z0F3SUJBZ0lJWDYrZHJBMDI0b1V3RFFZSktvWklodmNOQVFFTEJRQXdGVEVUTUJFR0ExVUUKQXhNS2EzVmlaWEp1WlhSbGN6QWVGdzB5TURBNE1EVXdOalUwTlRkYUZ3MHlNVEE0TURVd09ERXpORFJhTURZeApGekFWQmdOVkJBb1REbk41YzNSbGJUcHRZWE4wWlhKek1Sc3dHUVlEVlFRREV4SmtiMk5yWlhJdFptOXlMV1JsCmMydDBiM0F3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLQW9JQkFRQzFXU2FkQTY4ZFpFOW4Ka3BpbXRCS0Z2d2QwaWFFNWw0VDMrbDMyWXFGd3M2MU5zdnRKZXI0T0hNdFQxZHJ2S3RQbWlBL2xUYm1hZ1Y0KwpyZnVOZ2Frb0pOYVlzaFRhbkErdDNKaTBXOFNDem9PeUZWWWRwb0JjY0RNTGs3b0hSWE5pQXFqajdrNFpld3R4Cll5OXArbS9sN0JKTjZ1Y2tiWjRBYkpxcEIveUdacTc1cHBKQWw0RVBobkx3TUVLZTlkL1I2MThaLzVzQ2lXalUKYUNsRDN3V2NtTHl2Q0tzeUl4cVNLNndTV2ovNEZHQ1ZPamZMaDV4bGdydVZYTCsrWStnZ20zNDdHZnBQM2IvZgpYTjlnSWFSNUF2U1htd0RXMWx3Sk5OUEV5d2ZNL1MvU1pPQjV0ZHVmZjJ3NnBzMUNyMUl1a2NYOGJHdEs2TWw5ClNXSWNmTnVKQWdNQkFBR2pKekFsTUE0R0ExVWREd0VCL3dRRUF3SUZvREFUQmdOVkhTVUVEREFLQmdnckJnRUYKQlFjREFqQU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUFMSVN0dDVyZjNHWTJuRzZTVmVWZG9KNEsxWmw4WmJRagpJZUw0M1BhdjMzc1RmOGc4V3FGVUFycnU1ZzF1ZzJpR1pFNC9rUmVVU2xxK1pwMkVrZDJ0UHVMZXVIWExaNlBUCnpoUjh4M1dzMnNCR2lrdnFOa2MyZUcwcGFvRkkzbXd0VFdaeHc5U2FuT3oyd0thRVBLbkoyQmppczVPaDRGWnYKMUhLSllKeE5nY1F4NkZLL2U1VTU3eWF5MzIyNFpFYnNpQ2NXMHNqSEc1d01xWm85dzFXVTk3WXJZZGVkemVyNgpTekwweFA0bEM5T0loVjdIc3BlMFFyOGhvMXZHcGdWRk9UUjdaUFkyajk0VXZqRU1mdEZWOUxJRGVjZXhYcEZBCmJ6cy9jMXlCUUcrL1NneUxkd0FMT2J0S3BicUVMN24rT1RLSE9YQloxWjVUa2k1UkRXai9rZz09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
    client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBdFZrbW5RT3ZIV1JQWjVLWXByUVNoYjhIZEltaE9aZUU5L3BkOW1LaGNMT3RUYkw3ClNYcStEaHpMVTlYYTd5clQ1b2dQNVUyNW1vRmVQcTM3allHcEtDVFdtTElVMnB3UHJkeVl0RnZFZ3M2RHNoVlcKSGFhQVhIQXpDNU82QjBWellnS280KzVPR1hzTGNXTXZhZnB2NWV3U1Rlcm5KRzJlQUd5YXFRZjhobWF1K2FhUwpRSmVCRDRaeThEQkNudlhmMGV0ZkdmK2JBb2xvMUdncFE5OEZuSmk4cndpck1pTWFraXVzRWxvLytCUmdsVG8zCnk0ZWNaWUs3bFZ5L3ZtUG9JSnQrT3huNlQ5Mi8zMXpmWUNHa2VRTDBsNXNBMXRaY0NUVFR4TXNIelAwdjBtVGcKZWJYYm4zOXNPcWJOUXE5U0xwSEYvR3hyU3VqSmZVbGlISHpiaVFJREFRQUJBb0lCQUJhRmd0a2ZHd0ZRY3RnUApDYXd5VXRhUFdkS3NRUHlSQjY5d1JYa1MySy9SY0tWa28vYUZLdE0vMmpTSUs0OW1SemlyRHlaUUY4dkxZelB2CmtYMEtiRUtUQ0orZ0JoejhJcXBaNmhFRUVJeWFMeHJQc1lrdkkyUFhmMDlod2ttaTQyaDVvaFhrZzRpQWtuUTcKZUNqMk5DMWV1d3M0TUFxREhvczQ4YUFxWUw4a0RkbEVsb3liQWdLZDl2UG5uNkd5S01Pd3pDZUVZSzNUalV4SwpJSTl5emZJODc2aVJYczVkNmQwYlk3NURHQmxBVXMya01mMTgxN0pJWFA4bDNabnZQcm43NlFwejdtTTRVcE5FClZKdmN5UmE2MkdxT2tIZlFxZHdKcXJCREpYOGZqM1B1bVRkbmt6czdPdFNsZVZFNjErK0FBY0FHOXA1RUpzT3cKZUhyRG1QRUNnWUVBNVhSZXJtYmpINlpHOG9UTVVVMzFFSEE5N1lJTkFMV2szeTRWZEN2ZDFFZmk2NU5zejd0YQpmVGQ3YlhmbEE1MXcxeFM2NUhia1JiQThsYVFsTTJ6WGh5UTNSMUN0VDF2Tk4wNGsrVjkrZWVpUVY5WmY4VnVMCmVwNFBydEkrVzJONGs2bEkrY2cySkpmN1ZXc09WWW5NL05YdFdVdjdkbmlkc3c4MXkzcTFCQ1VDZ1lFQXlsUUsKMkRUZTZ2QUc3aW96WEpBUzhidkdLOG5ROFByU3d6YnUwTmxlaU1VeERneUNucXd6VkVGMGJMbWlLK1NjcmJTOQozZHlkNURhajd3b3lCU3RPUk5Qc0Y4UHRJUDViNXdqMiszcXFIbmhMSWpNSHNqWEdzUlY3RFd3blpwdS9XTldwClI2b1ZHVnZzTXI3YVRReVRxYm1sakl6d2N0RWpBOUNFd0xXWUNwVUNnWUFNeHBMekFQV0tjSmFvc0M0MEpMUU4KYTBKM2MvTkhZOVcvTzNDa1hFRG9yZXY3aytUT0dDOXNvMmN5N0NjTE1Ydno3Y3FPSi9TRDBORSt2N24yaUF2awpFV2NtaUY1dCtiMk9ZOEo1UkJEbHRzWTZyY3YvdzBvYjFLc28zN25yR3JtKzl3RUNMMXljWXYzeWZwaDR1TWg2CkYyczB1cDlRdEZ5TGVNQlZDUXNGSlFLQmdDZ2pIZmVnSFd6TDZ3UUtPU0F1c0tObDFiMVFEdlFQUGdwZDltaFcKQnN2cEduOG4raTJTQ2FpNFIybGVOZzZ4RG1malNCc2dMZTVjN0k1Nkhrd3FPQit0VHRPMEdYR0hKQ0lrQkNNRgpSWVEvNXQvRThjem5vQi8zRys4Z3Vudy9mMmtYRWJyV3dkTERDb3RTa3luZnlkZE1yY3ZCRmQ0aElPb2J6b29RClBEWTFBb0dCQUlkR2JmUTQ3WmFhaGhCSVB3eHUvaWtuVkx0S3JzRW5Qak1Cb1V3SDZUMW5CWGdZMzF3RncyTmIKTmI5dFZRYTk5aS9ic0dGeTNQYXFsYnN4V1VOSHZPczQ2WS9TQTRRVjFGcFpOeFJ5dTJNU0RWb2M1R0xKOElKSApMSkdZb3RIbkpXZnBTbFhaNG84bmRaVnk0TDZRbHZiL1M3OFlTVkZNRWl6NEJFS3ZjSUMzCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg==

```

**Additional context**
Trying to use lens with docker desktop community edition for windows with latest version: 2.3.0.4
docker desktop works fine with its own kubectl without any issue in native cmd terminal, but does not works with lens
ex:
kubectl version
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:26:26Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:18:29Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}

while lens is trying to use its own kubectl version:
C:\Users\user\AppData\Roaming\Lens\binaries\kubectl\1.16.8\kubectl.exe

Tried running lens  with "Run as Administrator" option still it does not work.

Sometimes i get the same error while connecting to cluster too from lens.
I am the administrator user of the windows os and have cluster-admin role rights on the kubernetes cluster i am trying to connect.

**Comment by trentondyck** [comment]
I get the same issue. Unfortunately something looks hard coded.  There are 2 files in AppData/roaming/lens/binaries/kubectl/1.15.11, .bash_set_path and .zlogin.  Both files are generated are hard coded to point to the 1.15.11 directory.  Even if you modify the contents of these files and change your path to program files or someplace else lens might be able to read from, the hard coded exe file doesn't read these files after the first time creating them.  
I have kubectl in my path, it works in git bash, the "simple" fix is this, give me an option to either set my own kube PATH environment variable, or an option to use a preconfigured system environment variable.  In other words, don't overwrite the "PATH" variable on startup...

**Comment by nevalla** [comment]
Fixed in #800
